docs/src/components/Layout/FrameworkChooser.tsx

```diff
@@ -1,66 +1,50 @@
-import {
-  Image,
-  ToggleButton,
-  ToggleButtonGroup,
-  VisuallyHidden,
-} from '@aws-amplify/ui-react';
+import { Flex, Image, Button, useTheme } from '@aws-amplify/ui-react';
 
+import { capitalize } from 'lodash';
 import { useCustomRouter } from '@/components/useCustomRouter';
+import Link from 'next/link';
 import { FRAMEWORKS } from '@/data/frameworks';
 
-export const FrameworkChooser = ({ platform }) => {
-  const { replace, pathname } = useCustomRouter();
+interface FrameworkLinkProps extends FrameworkChooserProps {
+  platform: string;
+}
 
-  const chooseFramework = (platform) => {
-    const { hash } = window.location;
-    replace(
-      {
-        hash,
-        pathname: pathname === '/' ? '/[platform]' : pathname,
-        query: { platform },
-      },
-      // `as?` prop  isn't needed when URL is already provided
-      undefined,
-      {
-        // Scroll to top if a new page
-        scroll: hash ? false : true,
-      }
-    );
+const FrameworkLink = ({ platform, onClick }: FrameworkLinkProps) => {
+  const { pathname, query } = useCustomRouter();
 
-    // Because layout may change, explicitly tell the browser to scroll to that anchor
-    // e.g. <a id="#variation" />
-    if (hash) {
-      document.getElementById(hash.slice(1)).scrollIntoView();
-    }
-  };
+  const isCurrent = query.platform === platform;
+  const classNames = `docs-framework-link ${isCurrent ? 'current' : ''}`;
```

Comment:
Nice to add `docs-framwork-link` to differentiate them from the mode switcher. We actually have a [ticket](https://app.asana.com/0/0/1202213232501039/f) about that. 

I wonder if we need to go even farther to add something like `docs-framework-link-${platform}`

Reply:
Would this be a class or should it be an id of `docs-framework-link-${platform}`? What are we using the class/id for?